src/features/duckplayer/util.js

```diff
@@ -225,11 +225,6 @@ export class VideoParams {
             return null
         }
 
-        // always exclude 'for rent'
-        if (url.searchParams.has('pp')) {
-            return null
-        }
```

Comment:
We think this was added to prevent overlays/clicks from occurring on movies - but as seen in the JSON changes, we can get around that with remote config changes instead.

Reply:
https://github.com/duckduckgo/privacy-configuration/pull/1294